resources/.eslintrc-standards--annoy-mode.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "rules": {
+        "no-magic-numbers": ["error", {                          // disallow magic numbers
+            "ignore": [0, 1, -1],
```

Comment:
I do often call out magic numbers in our styles, but there are a LOT of them in there. Far worse than our JS. Named variables and calculations are the direction I'm trying to move us towards though